tests/db_integration.rs

```diff
@@ -121,3 +129,112 @@ fn test_api_tokens() {
 
     clear_tables(db);
 }
+
+#[test]
+#[serial]
+fn test_package_versions() {
+    let db = &mut Database::default().conn();
+
+    // Set up session, user, token, and upload.
+    let session = db
+        .new_user_session(&mock_user_1(), 1000)
+        .expect("result is ok");
```

Comment:
nit: id simply unwrap in tests

Reply:
My personal preference is to use expects rather than unwraps in tests because I find it easier to pinpoint the errors, especially when there is more than one on a line.